apps/web-portal-client/README.md

```diff
@@ -0,0 +1,128 @@
+# web-portal-client
+
+
+## Start Project in Dev Mode
+
+`yarn start`
+
+
+## Design Decisions
+
+We choose to build our client on top of React + [Redux Tooltip](https://redux-toolkit.js.org/) stack
+
+Application consist within  `Features` each feature may has next layers
+
+1. `Slices` -  provides reducer and actions to manipulate within particular feature state slice
+1. `Selectors` - provides access to data required by a feature related to it's state slice
+1. `Effects` - contains interactions with side effects
+1. `Middlewares` - Redux middleware, main propose is to listen for app action flow and to some additional actions
+1. `Components` - 'dummy' components, which has no relations for `Slices` and `Selectors`
+1. `Containers` - components used by external features, usually use `Selectors` and Redux Actions
+
```

Comment:
could be some Models? or we will not allow the client to contain its own model layer and leave it on the server-side (make clean architecture)?

Reply:
I assume we are no need to use models on the client, yep on the server we would have some sort of models, it's an `access` layer